### PR TITLE
Creative bytes input

### PR DESCRIPTION
**Pull request by EmmaLouise2018**

### CODE REVIEW

File: docs/calculations.mdx [hunk 1]
```diff
@@ -132,32 +132,32 @@ Native product carousel
...
+| `region`                                   | The region of the country where the impression was delivered                                |
+| `channel`                                  | See [channels](#channels)                                                                   |
+| `device_type`                              | See [device types](#device-types)                                                           |
+| `network_type`                             | See [network types](#network-types)                                                         |
+| `property`                                 | See [property](#property)                                                                   |
+| `creative_ad_format`                       | Either a basic or vendor-provided ad format                                                 |
+| `creative_ad_platforms`                    | Ad serving, verification, and measurement ad platforms wrapping or embedded in the creative |
+| `creative_total_image_data_transfer_bytes` | Total bytes for the data transfer for the video itself , ideally measured by the CDN        |
```
Comment (ohalushchak-exadel) [inline docs/calculations.mdx:149]: for the `image` itself

File: docs/calculations.mdx [hunk 2]
```diff
@@ -361,15 +361,17 @@ loads = switch(ad_format.video_player.download_trigger):
...
 
 
-video_bytes = bitrate_kbps / 8 * 1024 x seconds_streamed x loads + default_video_vast_bytes
+video_bytes = (video_bitrate_kbps / 8 * 1024 x seconds_streamed + creative_video_vast_bytes) x loads
 ```
 
 For CTV/BVOD, use the power model:
 
 ```
+video_bitrate_kbps = creative_video_bitrate_kbps ?? default_video_bitrate_kbps[device_type]
```
Comment (EmmaLouise2018) [inline docs/calculations.mdx:370]: @ohalushchak-exadel

File: docs/calculations.mdx [hunk 3]
```diff
@@ -132,32 +132,32 @@ Native product carousel
...
+| `creative_ad_format`                       | Either a basic or vendor-provided ad format                                                 |
+| `creative_ad_platforms`                    | Ad serving, verification, and measurement ad platforms wrapping or embedded in the creative |
+| `creative_total_image_data_transfer_bytes` | Total bytes for the data transfer for the image itself , ideally measured by the CDN        |
+| `creative_image_sizes`                     | Array of image sizes included in the creative                                               |
+| `creative_total_audio_data_transfer_bytes` | Total bytes for the data transfer for the core audio asset, ideally measured by the CDN     |
+| `creative_audio_duration_seconds`          | Audio duration in seconds                                                                   |
+| `creative_total_video_data_transfer_bytes` | Total bytes for the data transfer for the core video asset, ideally measured by the CDN     |
+| `creative_video_vast_bytes`                | Size of VAST/VPAID wrapper for the video in bytes                                           |
```
Comment (bokelley) [inline docs/calculations.mdx:154]: should this be average? I guess same for the other metrics - should we clarify?

File: docs/calculations.mdx [hunk 4]
```diff
@@ -345,10 +345,10 @@ default_bitrate =
       is_primary_experience ?
       default_video_bitrate_kbps[device_type] :
       default_non_primary_video_bitrate_kbps
-bitrate =
+video_bitrate_kbps =
       creative_video_bitrate_kbps ??
       creative_video_size_bytes * 8 / 1024 / creative_video_duration  ??
-      default_bitrate_kbps
+      default_bitrate
```
Comment (bokelley) [inline docs/calculations.mdx:351]: why not kbps?

File: docs/calculations.mdx (hunk 3 above)
Comment (EmmaLouise2018) [reply docs/calculations.mdx:154]: Yeah, this is fair point. Will add into the descriptions

File: docs/calculations.mdx (hunk 4 above)
Comment (EmmaLouise2018) [reply docs/calculations.mdx]: The was actually another variable being used. Right fix was to fix the other variable to have `kbps` so now its logical

**Review by bokelley — APPROVED** [review]

**Pull request merged**